anthem/lyrics/loaders.py

```diff
@@ -51,7 +53,10 @@ def load_csv_stream(ctx, model_name, data, dialect='excel', encoding='utf-8',
     head = data.next()
     values = list(data)
     if values:
-        result = ctx.env[model_name].load(head, values)
+        model = ctx.env[model_name]
+        if context:
+            model = model.with_context(context)
```

Comment:
Ok I'll change that, wasn't sure if we wanted to keep a string or if passing a model would be more suited.